src/main/resources/assets/styles/_employeeList.scss

```diff
@@ -0,0 +1,163 @@
+.employees-list {
+	.row .container {
+		max-width: 1180px;
+
+		.page-title {
+			margin-bottom: 1.25rem;
+		}
+
+		.page-description p {
+			inline-size: 66%;
+			font-size: 20px;
+			line-height: 32px;
+		}
+	}
+
+	.banner {
+		height: 304px;
```

Comment:
Denne høyden bør ikke være fast, kanskje ha en min-height, men ellers sørge for at det er 80 px under ingress.